rllib/examples/two_trainer_workflow.py

```diff
@@ -112,8 +111,9 @@ def training_step(self) -> ResultDict:
 
         # DQN sub-flow.
         dqn_train_results = {}
-        dqn_train_batch = self.local_replay_buffer.sample(num_items=64)
-        if dqn_train_batch is not None:
+
+        if self._counters[NUM_ENV_STEPS_SAMPLED] > 1000:
```

Comment:
This is sort of a manual implementation of the learning starts logic. Since this example implements custom training logic, we put it here in a simplified verison.

Reply:
thanks for the clarification. I really like how you provide these extra PR explanations that answer most likely confusions preemptively. 👏